SimpleArraySum.cs

```diff
@@ -0,0 +1,47 @@
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+    public static int simpleArraySum(List<int> ar)
+    {
+      int sum = 0;
+      Int32 length = ar.Count;
```

Comment:
No need to use extra variable. You can directly use ar.Count in for loop

Reply:
@narayanagunturi I have resolved this problem.